prairielib/lib/migrations.sql

```diff
@@ -5,7 +5,9 @@ CREATE TABLE IF NOT EXISTS migrations (
   index INTEGER,
   project TEXT DEFAULT 'prairielearn',
   applied_at TIMESTAMP WITH TIME ZONE,
-  UNIQUE(project, index)
+  timestamp TEXT,
+  UNIQUE(project, index),
+  UNIQUE(project, filename)
```

Comment:
This change wasn't a bug, but it did result in an extraneous log being printed when populating a fresh database.